src/SuperSelectField.js

```diff
@@ -515,7 +531,7 @@ class SelectField extends Component {
       : elementHeight
     */
     const autoCompleteHeight = this.state.showAutocomplete ? 53 : 0
-    const footerHeight = menuCloseButton ? 36 : 0
+    const footerHeight = (menuCloseButton || menuCancelButton) ? 36 : 0
```

Comment:
What are the difference(s) between the two ? i smell redundancy...

Reply:
One is submiting changes, other is canceling and reseting changes

Reply:
OMG, i totally misunderstood you:
- in your original issue, i thought you wanted some button deselecting all selected options, SSF returning an empty value
- but your last comment here sound more like you wish for a canceling button on a per-option basis, ie deselect selected options from some queue, one by one ?

Reply:
About deselect all - this is other feature request not implemented here

Reply:
Honestly please stay in your issue topic, not in a PR. I think more and more data manipulation is NOT SSF duty, but dev's....
We are currently going into a big environment transfer (see PR). So didnot get time yet to help you with this.